airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -15,14 +15,17 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from contextlib import closing
 from typing import Any, Dict, Optional, Tuple
+from io import StringIO
```

Comment:
```suggestion
from io import StringIO
from typing import Any, Dict, Optional, Tuple
```

Reply:
This should fix the failing static test: https://github.com/apache/airflow/pull/11350/checks?check_run_id=1973389981#step:8:244